lib/registration.pm

```diff
@@ -172,6 +172,9 @@ sub fill_in_registration_data {
         if (get_var('SCC_URL') || get_var('SMT_URL')) {
             push @tags, 'untrusted-ca-cert';
         }
+        # The "Extension and Module Selection" won't be shown during upgrade to sle15, refer to:
+        # https://bugzilla.suse.com/show_bug.cgi?id=1070031#c11
```

Comment:
if this is workaround due to the bug you need to add softfilure

Reply:
No, this is not a workaround, I thought it was a bug that "Extension and Module Selection" not shown during upgrade to sle 15. But it turns out to be expected behavior according to https://bugzilla.suse.com/show_bug.cgi?id=1070031#c11